README.md

```diff
@@ -141,8 +141,9 @@ See [remarkable.guide](https://remarkable.guide/tech/factory-reset.html) for mor
 - [paginator](https://github.com/aflusche/paginator) - Physical foot pedal to turn pages on the device with no hands (e.g. for playing sheet music).
 - [ReCept](https://github.com/funkey/recept/) - Fix for the rM2 jagged line issue.
 - [reLuminate](https://github.com/unreMarkableLabs/reLuminate) - Enable enhanced screen brightness levels on the reMarkable Paper Pro.
-- [rM-signature-patch](https://github.com/Barabazs/rM-signature-patch) - Simple script to remove that pesky advert at the bottom of a mail originating from a reMarkable.
 - [remarvin](https://github.com/plan5/remarvin) - Profile and file encrytion manager that allows to manage notebooks for different users and to optionally protect the files with a password through gocryptfs-based encryption (device only).
+- [rM-signature-patch](https://github.com/Barabazs/rM-signature-patch) - Simple script to remove that pesky advert at the bottom of a mail originating from a reMarkable.
```

Comment:
Shouldn't `rM-signature-patch` come after `rmtree` when sorted, as `-` comes after `t`?

Reply:
I don't think we've defined how it would be sorted yet. I'm fine with using your idea of sorting based on the integer value of the character, which for both ASCII and UTF-8 puts - before a.